src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { closeDbConnection, createConnection, migrate } from './db/eventDb'
+import { pollEvents, todaysEvents } from './services/eventsService'
+import { foodListByRestaurant } from './services/foodlistService'
+import { sendMessage } from './services/telegramService'
+import { JobMode } from './types'
+
+exports.handler = async ({ jobMode }: { jobMode: JobMode }) => {
+  await migrate()
+  await createConnection()
+
+  try {
+    switch (jobMode) {
+      case 'postFood':
+        await sendMessage('daily', await foodListByRestaurant('exactum'), false)
```

Comment:
I think it would be nicer to replace the embedded await with ` await foodListByRestaurant('exactum').then(sendMessage)`